lib/ddtrace/profiling/preload.rb

```diff
@@ -0,0 +1,7 @@
+require 'ddtrace/profiling'
+
+if Datadog::Profiling.supported? && Datadog::Profiling.native_cpu_time_supported?
+  Datadog::Profiling::Tasks::Setup.new.run
+else
+  puts '[DDTRACE] Profiling not supported; skipping preload.'
```

Comment:
`puts` sounds you're always printing that to `stdout`. Isn't that too verbose for an internal detail?

Reply:
It's more of a warning at program startup... yeah it might get buried in the rest of STDOUT, but there's no mechanism for logging this early in the Ruby application for the Ruby trace library right now (because configuration is done in `.rb` files later in startup, not via ENV or YML.) It would be better to write this to some pre-configured log file (determined via ENV or YML) in the future.